backends/cuda-ref/ceed-cuda-ref-basis.c

```diff
@@ -38,7 +36,7 @@ int CeedBasisApply_Cuda(CeedBasis basis, const CeedInt num_elem, CeedTransposeMo
   CeedCallBackend(CeedVectorGetArrayWrite(v, CEED_MEM_DEVICE, &d_v));
 
   // Clear v for transpose operation
-  if (t_mode == CEED_TRANSPOSE) {
+  if (transpose) {
```

Comment:
I haven't noticed this before, but our style would usually be `is_transpose` or something like that for bools. At some point I should check the GPU backends for variable naming conventions, but its not important here.

Reply:
Sounds good to leave it to a later PR where it can be addressed everywhere (sycl, occa backends both have `if (transpose)` statements in my quick search).